jedi/plugins/stdlib.py

```diff
@@ -790,10 +861,37 @@ def wrapper(cls, metaclasses):
                 return [DictFilter({
                     name.string_name: EnumInstance(cls, name).name for name in filter_.values()
                 })]
+
+            if metaclass.py__name__() == 'ModelBase' \
+                    and metaclass.get_root_context().py__name__() == 'django.db.models.base':
+                django_dict_filter = new_django_dict_filter(cls)
+                if django_dict_filter is not None:
+                    return django_dict_filter
+
         return func(cls, metaclasses)
     return wrapper
 
 
+class DjangoModelField(LazyValueWrapper):
+    def __init__(self, cls, name):
+        self.inference_state = cls.inference_state
+        self._cls = cls  # Corresponds to super().__self__
+        self._name = name
+        self.tree_node = self._name.tree_name
+
+    @safe_property
+    def name(self):
+        return ValueName(self, self._name.tree_name)
+
+    def _get_wrapped_value(self):
+        obj, = self._cls.execute_with_values()
+        return obj
+
+    def get_filters(self, origin_scope=None):
+        for f in self._get_wrapped_value().get_filters():
+            yield f
```

Comment:
AFAIK you don't need to define `get_filters` here, that's pretty much what it already does if you don't implement it.